.github/workflows/test.yml

```diff
@@ -136,7 +147,9 @@ jobs:
         - x86_64-unknown-linux-gnu
         - i686-unknown-linux-gnu
         - aarch64-unknown-linux-gnu
-        - armv7-unknown-linux-gnueabihf
+        # Disabled because it fails in GitHub Actions:
+        # `qemu: uncaught target signal 11 (Segmentation fault) - core dumped`
+        # - armv7-unknown-linux-gnueabihf

```

Comment:
This is a potentially contentious change. Perfect may be the enemy of good here, though: getting most tests running again is probably worth it.

Reply:
Does this also fail when using a newer Rust version? If it's unrelated to that then I agree that removing this platform is the best choice, as there's no point in running CI on a platform that always fails.

Reply:
This may not be enough, sadly. Another test run for `aarch64-unknown-linux-gnu` showed:

```
qemu-aarch64: QEMU internal SIGSEGV {code=MAPERR, addr=0x20}
```

> fail when using a newer Rust version

The test for `armv7-unknown-linux-gnueabihf` failed on the beta channel, the test for `aarch64-unknown-linux-gnu` failed on nightly.